CartPage/cartPage.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/style.css" />
+    <script
+      src="https://kit.fontawesome.com/e06444868d.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo">
+        <h2>Nike</h2>
+      </div>
+
+      <input class="search-bar" type="text" placeholder="Search" />
+      <nav>
+        <button class="btn margin-right-xxs">Login</button>
+        <i class="fas fa-shopping-cart fa-lg"></i>
+      </nav>
+    </header>
+    <div class="cart-container">
+      <div class="horizontal-ecomm-card">
+        <div class="horizontal-card-image-bg">
+          <img
+            class="card-image"
+            src="./jordan-zoom-separate-pf-basketball-shoes-kGPg2r-removebg.png"
+            alt=""
```

Comment:
Suggestion to add an alt attribute to the image as this will be helpful when or if image does not load

Reply:
Done! Thank you Raksha for the suggestions.